doc/userguide/rules/payload-keywords.rst

```diff
@@ -516,14 +516,15 @@ The ``byte_jump`` keyword allows for the ability to select a ``<num of bytes>``
 
 Format::
 
-  byte_jump:<num of bytes>, <offset> [, relative][, multiplier <mult_value>] \
+  byte_jump:<num of bytes> | <variable-name>, <offset> [, relative][, multiplier <mult_value>] \
```

Comment:
would it make sense to extend this to others too? multiplier, post_offset, bitmask?

Reply:
@jmtaylor90  Thoughts on whether variables would be useful elsewhere?

`bitmask` has odd semantics so i don't think we'd want to include that

Reply:
Talking with the team a bit, it would be nice to have variable use available wherever possible.

Reply:
I suggest we extend variable support to other elements in a different set of tickets for each of the `byte_*` keywords)

Reply:
Definitely fine by me, don't want to unnecessarily scope creep on you :)